xbmc/network/TCPServer.cpp

```diff
@@ -50,7 +50,7 @@ static const bdaddr_t bt_bdaddr_local = {{0, 0, 0, 0xff, 0xff, 0xff}};
 
 using namespace JSONRPC;
 
-#define RECEIVEBUFFER 1024
+#define RECEIVEBUFFER 4096
```

Comment:
please use an anonymous namespace to define the maxBufferLength which is preferred over static
```c
namespace
{
constexpr size_t maxBufferLength = 64 * 1024;
}
```